docs/_documentations/remotedeploy-eclipse.md

```diff
@@ -45,7 +45,8 @@ The IDE and Codewind are connected.
 
 # 2. Add a deployment registry
 
-Before projects can be deployed on Kubernetes, you must specify a Docker push registry. This example uses Docker Hub. 
+Before projects can be deployed on Kubernetes, you must specify a Docker push registry. This example uses Docker Hub.
+For more information about registries, see [Adding an image registry in remote Codewind](remote-setupregistries.md).
```

Comment:
I think the link should go to `Codewind container registry guidance` on that page since there are already instructions on how to add a registry in this document. Same for VSCode.